chapters/publication.tex

```diff
@@ -1,96 +1,171 @@
 %------------------------------------------------
 
 \begin{fullwidth}
-Increasingly, research assistants are relied on to manage some or all
-of the publication process. This can include
-managing the choice of software,
-coordinating referencing and bibliography,
-tracking changes across various authors and versions,
-and preparing final reports or papers for release or submission.
-Modern software tools can make a lot of these processes easier.
-Unfortunately there is some learning curve,
-particularly for lead authors who have been publishing for a long time.
-This chapter suggests some tools and processes
-that can make writing and publishing in a team significantly easier.
-It will provide resources
-to judge how best to adapt your team to the tools you agree upon,
-since all teams vary in composition and technical experience.
-
+Publishing academic research today extends well beyond writing up a Word document alone.
```

Comment:
Publishing cannot even in the past only have consisted of writing a document. 
```suggestion
Publishing academic research today extends well beyond writing up and submitting a Word document alone.
```

Reply:
cd278a7d7aaf2666da6dd77901f8d556fff53c55